Purchasing.Mvc/Views/Order/_OrderDetails.cshtml

```diff
@@ -56,7 +56,7 @@
                     <h2>Select an approver &amp; account manager</h2>
                     <div class="editor-label">Select An Approver:</div>
                     <div class="editor-field">
-                        @this.Select("approvers").FirstOption("--Select An Approver (Optional)--").Options(Model.Approvers, x => x.Id, x => x.FullNameAndId)
+                        @Html.Select("approvers", o => o.FirstOption("--Select An Approver (Optional)--").Options(Model.Approvers, x => x.Id, x => x.FullNameAndId))
```

Comment:
If we want this to order correctly, do it like the account managers select list below.
FirstOption isn't making it the first option here

Reply:
Nah, just a quick fix in `HtmlHelperSelectExtensions.cs`